app/src/main/java/com/halim/adam/cashmaster/TransactionHistory.java

```diff
@@ -1,16 +1,38 @@
 package com.halim.adam.cashmaster;
 
+import android.app.Activity;
 import android.content.Intent;
-import android.support.v7.app.AppCompatActivity;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
+import android.widget.TextView;
 
-public class TransactionHistory extends AppCompatActivity {
+public class TransactionHistory extends Activity {
+    protected Cursor cursor;
+    TextView transactionList;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_transaction_history);
+        transactionList = (TextView) findViewById(R.id.transactionList);
+
+        DatabaseHelper dbHelper = new DatabaseHelper(this);
+        dbHelper.InsertIncome("Baby", 800);
+
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
```

Comment:
This block can be moved to its own function inside the dbHelper class so the SQL logic does not leak to here. For example 

```java
dbHelper.getIncomeList();
```